storage/storage-interface/src/lib.rs

```diff
@@ -55,7 +55,7 @@ pub use executed_trees::ExecutedTrees;
 
 // This is last line of defense against large queries slipping through external facing interfaces,
 // like the API and State Sync, etc.
-pub const MAX_REQUEST_LIMIT: u64 = 10000;
+pub const MAX_REQUEST_LIMIT: u64 = 20_000;
```

Comment:
Comment sounds scary :) I guess we need storage team signoff

Reply:
Yeah, it should be fine. This is a legacy number 😄